### ISSUE TITLE
Incompatible with string array column types?

### ISSUE THREAD

**Issue by robwebdev** [markusahlstrand/knex-data-api-client#76]
Hi, thanks for the module.

I've run into an issue using the postgres clients with certain column types created with `t.specificType`

We have some columns created like the example below

```
export async function up(knex: Knex): Promise<void> {
  return knex.schema.alterTable('files', t => {
    t.specificType('errors', 'text ARRAY')
      .notNullable()
      .defaultTo('{}')
  })
}
```

When querying this column the data is returned in the following structure:

```
{errors: {stringValue: []}}
```

rather than the expected `[]`

Does that mean this client is incompatible with these types of columns?

**Comment by markusahlstrand** [comment]
HI @robwebdev,

I haven't actually used the specificType before. I think we need to create a new test with the expected behaviour.

**Comment by markusahlstrand** [comment]
Added a test a what I hope is a fix for this issue :) It is released as version 1.6.2 now.

**Comment by robwebdev** [comment]
oh wow thanks for jumping on this @markusahlstrand. I'll give this test in the real world again soon but the test you added looks like it will fix things our issue. 😎
